src/toil/fileStore.py

```diff
@@ -851,7 +851,8 @@ def cleanCache(self, newJobReqs):
                 #                  (self.decodedFileID(cachedFile), cachedFileSize))
                 logger.debug('CACHE: Evicted  file with ID \'%s\' (%s bytes)' %
                              (self.decodedFileID(cachedFile), cachedFileSize))
-            assert cacheInfo.isBalanced(), 'Unable to free up enough space for caching.'
```

Comment:
I'm going to take a guess and say you're using singleMachine? This won't happen after the disk tracking we just put into Toil